SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -78,6 +86,9 @@ tar xf %{SOURCE1}
 mkdir -p .cargo
 cp %{SOURCE2} .cargo/
 %endif
+# The vendored archive has been populated based on the patch, so we need to
+# repatch here as well in order to use the same versions
+%autopatch -p1
```

Comment:
One thing I noticed right now: looks like in dev we've added `0001-hypervisor-mshv-Fix-panic-when-rejecting-extended-gu.patch` and along with that the `-p1` flag to `%autosetup`. This will now conflict with `%autopatch -p1` added in fast-track. One way to solve it would be to remove `-p1` from `%autosetup` and keep the `%autopatch` one, since we need to patch **after** we extract the vendored source.